Objects/longobject.c

```diff
@@ -4261,6 +4347,26 @@ long_pow(PyObject *v, PyObject *w, PyObject *x)
             goto Done;
         }
 
+        /* if exponent is negative, negate the exponent and
+           replace the base with a modular inverse */
+        if (Py_SIZE(b) < 0) {
+            temp = (PyLongObject *)_PyLong_Copy(b);
```

Comment:
I think `Py_SETREF()` should be used in this block.

Reply:
I'd rather keep it the way it is in this PR, for consistency with the existing previous block that negates the modulus. I'm happy to make a separate PR that updates both blocks (and other uses in `longobject.c`) to use `Py_SETREF`, but I don't want the `long_pow` code to be doing the same thing in two different ways in two places, and I want to keep this particular PR focused on the inversion feature.